### ISSUE TITLE
Insert script crash generates server termination

### ISSUE THREAD

**Issue by jinxmcg** [akumuli/Akumuli#252]
Hi,

I was trying to write some test metrics. and my insert script crashed. should it terminate the server too?
(i started it with docker)

Any idea?

```
2018-03-09 17:14:07,252 resp-protocol-parser [INFO] Starting protocol parser
2018-03-09 17:14:07,262 Main [ERROR] Invalid session cache, id = 1979
2018-03-09 17:14:07,262 Main [ERROR] Terminating (core dumped)
```

Thank you

**Comment by Lazin** [comment]
Hi,
Can you share the log file and your insert script or the data that you're inserting (at least the sample)?

**Comment by jinxmcg** [comment]
Hi, 

Here is the log:
[akumuli.log](https://github.com/akumuli/Akumuli/files/1802553/akumuli.log)

Let me explain the test I did. I was trying to see the ingestion speed and to check if it is a good fit for my use case: Aggregate access logs. sum traffic and no of requests grouped by a specific domain and/or specific user (i still do not know yet as cardinality of the domains is pretty high)

So all the data is ingested at random because this is a test. The ingested volume was about 50000 points in total using TCP and RESP, when I did a mistake and my script crashed (script was written in go)...also i did not close the TCP connection 

```
package main

import (
	"bytes"
	"io/ioutil"
	"math/rand"
	"net"
	"net/http"
	"os"
	"strconv"
	"time"
)

//docker run -d -p 8181:8181 -p 8282:8282 -p 8383:8383 -p 4242:4242 -v /tmp/akumuli:/Akumuli -e nvolumes='0' -e volume_size='4GB' akumuli/akumuli:skylake

func main() {
	servAddr := "127.0.0.1:8282"
	tcpAddr, err := net.ResolveTCPAddr("tcp", servAddr)
	if err != nil {
		println("ResolveTCPAddr failed:", err.Error())
		os.Exit(1)
	}

	conn, err := net.DialTCP("tcp", nil, tcpAddr)
	if err != nil {
		println("Dial failed:", err.Error())
		os.Exit(1)
	}

	domains := [...]string{"www.google.com", "www.yahoo.com", "www.youtube.com"}

	now := time.Now()
	tmp := strconv.FormatInt(now.UnixNano(), 10)
	println("Time:", tmp)
	rows := 0
	for i := 0; i < 10000; i++ {
		now = time.Now()
		tmp = strconv.FormatInt(now.UnixNano(), 10)
		user := rand.Intn(13)
		traffic := rand.Int()
		server := rand.Intn(126)
		domain := domains[rand.Intn(len(domains)-1)]

		row := "+access user=" + strconv.Itoa(user) + " host=srv" + strconv.Itoa(server) + " domain=" + domain + "\r\n+" + tmp + "\r\n:" + strconv.Itoa(traffic) + "\r\n"
		_, err = conn.Write([]byte(row))
		if err != nil {
			println("Write to server failed:", err.Error())
			os.Exit(1)
		}
		rows++
	}
	println("write to server ", rows)

	/*
	   //wait for answer...in a successful write no answer comes
	   reply := make([]byte, 1024)
	   _, err = conn.Read(reply)
	   if err != nil {
	       println("Write to server failed:", err.Error())
	       os.Exit(1)
	   }
	   println("reply from server=", string(reply))

	   conn.Close()
	*/

	url := "http://127.0.0.1:8181/api/query"
	println("URL:>", url)
	var jsonStr = []byte(`
    {
	"group-aggregate":{
	    "metric":"access",
	    "step":"1m",
	    "func":["count"]
	},
	"where" : {
	    "domain":"www.yahoo.com",
	    "user": "13"
	},
	"range":{
	    "from":"1520612217943862700",
	    "to":"` + tmp + `"
	},
	"output": { "format": "csv" }
    }`)
	//	"group-by": ["user"],
	/*
		"aggregate": {"access":"sum"}

	*/

	resp, err := http.Post(url, "application/x-www-form-urlencoded", bytes.NewBuffer(jsonStr))
	if err != nil {
		panic(err)
	}
	defer resp.Body.Close()

	//println("response Status:", resp.Status)
	body, _ := ioutil.ReadAll(resp.Body)
	println("response Body:\n", string(body))

}
```

**Comment by Lazin** [comment]
I created the fix but it's not in the master yet (this is the PR - https://github.com/akumuli/Akumuli/pull/254)
May I ask you to check the latest image from [this repository](https://hub.docker.com/r/akumuli/test/tags/)? When the build finishes it will upload new images to both generic and skylake tags (this should happen in 40 minutes).
I checked your script. On my machine it writes the data correctly.
